src/components/bottom/ResultList.tsx

```diff
@@ -0,0 +1,76 @@
+import { Component } from 'react';
+import { getData, type Species } from '../../utils/api';
+import style from './styles.module.scss';
+import spinner from '../../assets/react.svg';
+
+interface Props {
+  value: string;
+}
+
+export class ResultList extends Component<Props> {
+  state: {
+    result: Species[];
+    loading: boolean;
+  };
+  key = 0;
```

Comment:
element id can be used as a key, no need to generate extra keys, because they don't really match with existing list items. Key property is designed to track what list items shouldn't be re-rendered in case of list update